xbmc/network/WebServer.cpp

```diff
@@ -765,6 +766,10 @@ int CWebServer::CreateFileDownloadResponse(const std::shared_ptr<IHTTPRequestHan
   std::shared_ptr<XFILE::CFile> file = std::make_shared<XFILE::CFile>();
   std::string filePath = handler->GetResponseFile();
 
+  // white/black list access check
+  if (!CFileUtils::ZebraListAccessCheck(filePath))
+    return SendErrorResponse(request.connection, MHD_HTTP_NOT_FOUND, request.method);
```

Comment:
I'm with @MrMC here, 403 could leak info to a would be attacker that the file is there so a 404 is safer in that regard.